pyproject.toml

```diff
@@ -15,36 +15,36 @@ meta-lookup = 'main:main'
 
 [tool.poetry.dependencies]
 python = "^3.9.1"
-fastapi = "^0.65.0"
-uvicorn = "^0.13.3"
-requests = "^2.24.0"
+fastapi = "^0.65"
+uvicorn = "^0.13"
+requests = "^2.27"
 adblockparser = "^0.7"
-lxml = "^4.7.1"
-tldextract = "^3.0.2"
+lxml = "^4.9"
+tldextract = "^3.3"
 "pdfminer.six" = "^20201018"
-PyPDF2 = "^1.27.12"
+PyPDF2 = "^1.28"
 aiohttp = "^3.8"
-beautifulsoup4 = "^4.10.0"
-databases = { extras = ["postgresql","sqlite"], version = "^0.5.1" }
-python-dotenv = "^0.20.0"
+beautifulsoup4 = "^4.11"
+databases = { extras = ["postgresql","sqlite"], version = "^0.5" }
+python-dotenv = "^0.20"
 
 [tool.poetry.dev-dependencies]
-pylint = "^2.6.0"
-pre-commit = "^2.10.0"
-pytest = "^6.2.2"
-pytest-cov = "^2.11.1"
-pytest-mock = "^3.5.1"
-pandas = "^1.2.1"
-altair = "^4.1.0"
-altair-viewer = "^0.3.0"
-mkdocs = "^1.1.2"
-pip-licenses = "^3.3.0"
-wily = "^1.19.0"
-vulture = "^2.3"
-pyinstrument = "^3.3.0"
-pytest-asyncio = "^0.18.3"
-locust = "^2.9.0"
-httpx = "^0.23.0"
+pylint = "^2.14"
+pre-commit = "^2.19"
+pytest = "^6.2"
+pytest-cov = "^2.12"
+pytest-mock = "^3.7"
+pandas = "^1.4"
+altair = "^4.2"
```

Comment:
Nice - but I cannot find a usage of altair anymore. Could you check, please, if they can be deleted?

Reply:
removed dev dependencies:
 - altair
 - altair-viewer
 - pandas
 - wily
 - vulture
 - pip-licenses
 - pyinstrument